x/emissions/keeper/params_test.go

```diff
@@ -0,0 +1,213 @@
+		},
+		{
+			name: "validator emission percentage too low",
+			params: emissionstypes.Params{
+				MaxBondFactor:               "1.25",
+				MinBondFactor:               "0.75",
+				AvgBlockTime:                "6.00",
+				TargetBondRatio:             "00.67",
+				ValidatorEmissionPercentage: "-1.50",
+				ObserverEmissionPercentage:  "00.25",
+				TssSignerEmissionPercentage: "00.25",
+				DurationFactorConstant:      "0.001877876953694702",
+			},
+			isPanic: "validator emission percentage cannot be less than 0 percent",
+		},
+		{
+			name: "observer percentage too low",
+			params: emissionstypes.Params{
+				MaxBondFactor:               "1.25",
+				MinBondFactor:               "0.75",
+				AvgBlockTime:                "6.00",
+				TargetBondRatio:             "00.67",
+				ValidatorEmissionPercentage: "00.50",
+				ObserverEmissionPercentage:  "-00.25",
+				TssSignerEmissionPercentage: "00.25",
+				DurationFactorConstant:      "0.001877876953694702",
+			},
+			isPanic: "observer emission percentage cannot be less than 0 percent",
+		},
+		{
+			name: "observer percentage too high",
+			params: emissionstypes.Params{
+				MaxBondFactor:               "1.25",
+				MinBondFactor:               "0.75",
+				AvgBlockTime:                "6.00",
+				TargetBondRatio:             "00.67",
+				ValidatorEmissionPercentage: "00.50",
+				ObserverEmissionPercentage:  "150.25",
+				TssSignerEmissionPercentage: "00.25",
+				DurationFactorConstant:      "0.001877876953694702",
+			},
+			isPanic: "observer emission percentage cannot be more than 100 percent",
+		},
+		{
+			name: "tss signer percentage too high",
+			params: emissionstypes.Params{
+				MaxBondFactor:               "1.25",
+				MinBondFactor:               "0.75",
+				AvgBlockTime:                "6.00",
+				TargetBondRatio:             "00.67",
+				ValidatorEmissionPercentage: "00.50",
+				ObserverEmissionPercentage:  "00.25",
+				TssSignerEmissionPercentage: "102.22",
+				DurationFactorConstant:      "0.001877876953694702",
+			},
+			isPanic: "tss emission percentage cannot be more than 100 percent",
+		},
+		{
+			name: "tss signer percentage too loo",
+			params: emissionstypes.Params{
+				MaxBondFactor:               "1.25",
+				MinBondFactor:               "0.75",
+				AvgBlockTime:                "6.00",
+				TargetBondRatio:             "00.67",
+				ValidatorEmissionPercentage: "00.50",
+				ObserverEmissionPercentage:  "00.25",
+				TssSignerEmissionPercentage: "-102.22",
+				DurationFactorConstant:      "0.001877876953694702",
+			},
+			isPanic: "tss emission percentage cannot be less than 0 percent",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			k, ctx := keepertest.EmissionsKeeper(t)
+			defaultParams := k.GetParams(ctx)
+			require.NotEqualf(t, tt.params, defaultParams, "expected: %v, got: %v", tt.params, defaultParams)
+			assertPanic(t, func() {
```

Comment:
You normally have a `Panics` utility as part of `testify` library

Reply:
Yes, but I could not get asser. Panic to check for Error.Contains, it does an equal string match, which was not working as I would want it to. 
Please feel free to let me know if you know of how to get it to work , I would be happy to make the change